internal/app/public.go

```diff
@@ -39,6 +39,34 @@ func publicAPI(
 		return c.JSON(http.StatusOK, output)
 	})
 
+	r.POST("/password-reset-request", func(c echo.Context) error {
+		var inp http1.PasswordResetRequestInput
+		if err := c.Bind(&inp); err != nil {
+			return &echo.HTTPError{Code: http.StatusBadRequest, Message: fmt.Errorf("failed to parse request body: %w", err)}
```

Comment:
As you can see https://github.com/labstack/echo/blob/master/bind.go#L59, errors do not need to be wrapped with echo.HTTPError (they are already wrapped)

```suggestion
			return err
```